### PR TITLE
Fixing `timesheet`

### PR DESCRIPTION
**Pull request by KINFOO**
# Bugs in previous code

1. Notifications were fired daily
1. Bot was disconnected by slack, because all notifications were fired simultaneously.

# Solutions

## Notifications the right days

Update `node-schedule` to `1.1.x` to enjoy `RecurrenceRule.day` ranges:

```js
var rule = new schedule.RecurrenceRule();
rule.date = [27, 29];
```

## Soft notifications sending

Notifications are sent within a _2 minutes_ window.

```js
var rule = new schedule.RecurrenceRule();
// ...
rule.minute = _.random(29, 31);
rule.second = _.random(0, 59);
```

# Testing

Upgrading `node-schedule` is not supposed to affect `planning`.

Anyhow, for safety, I'll run this PR version with `Air2d2` tomorrow.
:pray:

### CODE REVIEW

**Comment by KINFOO** [comment]
# Testing

It Works

## Timesheet fix

j9hico13ox

## Planning compatibility

We did not receive any planning, it think it is a _JIRA credentials_ thing.
Here is proof.

With this code in  `planning.js`:

```js
    var rule = new schedule.RecurrenceRule();
    rule.dayOfWeek = [new schedule.Range(1, 5)];
    rule.hour = 11;
    rule.minute = 51;
```

Logs are

    [Fri Jul 08 2016 11:51:00 GMT+0200 (CEST)] DEBUG Build planning task for: air2d2
    [Fri Jul 08 2016 11:51:00 GMT+0200 (CEST)] DEBUG Build planning task for: airbot
    [Fri Jul 08 2016 11:51:00 GMT+0200 (CEST)] DEBUG Build planning task for: alfredo
    [Fri Jul 08 2016 11:51:00 GMT+0200 (CEST)] DEBUG Build planning task for: arenoux
    [Fri Jul 08 2016 11:51:00 GMT+0200 (CEST)] DEBUG Build planning task for: arielle
    [Fri Jul 08 2016 11:51:00 GMT+0200 (CEST)] DEBUG Build planning task for: bmiegemolle
    [Fri Jul 08 2016 11:51:00 GMT+0200 (CEST)] DEBUG Build planning task for: camillahenrysson
    [Fri Jul 08 2016 11:51:00 GMT+0200 (CEST)] DEBUG Build planning task for: christophe
    [Fri Jul 08 2016 11:51:00 GMT+0200 (CEST)] DEBUG Build planning task for: clam
    [Fri Jul 08 2016 11:51:00 GMT+0200 (CEST)] DEBUG Build planning task for: dfrancois
    [Fri Jul 08 2016 11:51:00 GMT+0200 (CEST)] DEBUG Build planning task for: dlahay
    [Fri Jul 08 2016 11:51:00 GMT+0200 (CEST)] DEBUG Build planning task for: dpo
    [Fri Jul 08 2016 11:51:00 GMT+0200 (CEST)] DEBUG Build planning task for: druant
    [Fri Jul 08 2016 11:51:00 GMT+0200 (CEST)] DEBUG Build planning task for: dsciamma
    [Fri Jul 08 2016 11:51:00 GMT+0200 (CEST)] DEBUG Build planning task for: duty
    [Fri Jul 08 2016 11:51:00 GMT+0200 (CEST)] DEBUG Build planning task for: fcouadau
    ...

So task is triggered as we want

## Long story short

@sebz  :+1:

**Comment by sebz** [comment]
LGTM :+1:
